drivers/display/display_stm32_ltdc.c

```diff
@@ -267,13 +263,9 @@ static int stm32_ltdc_display_blanking_off(const struct device *dev)
 static int stm32_ltdc_display_blanking_on(const struct device *dev)
 {
 	const struct display_stm32_ltdc_config *config = dev->config;
-	const struct device *display_dev = config->display_controller;
+	const struct device *display_dev = config->panel_controller;
 
-	if (display_dev == NULL) {
-		return 0;
-	}
-
-	if (!device_is_ready(config->display_controller)) {
+	if (!device_is_ready(display_dev)) {
 		LOG_ERR("Display device %s not ready", display_dev->name);
```

Comment:
Can't use `display_dev->name` (`display_dev` may be NULL).

Also, nit:
```suggestion
		LOG_ERR("Panel device %s not ready", display_dev->name);
```

Reply:
The `device_is_ready` already checks for `null`:
```
	if (dev == NULL) {
		return false;
	}
```

Reply:
If `display_dev` is NULL, then `device_is_ready(display_dev)` returns false, and the `if` block is executed.